homeassistant/components/stream/core.py

```diff
@@ -155,14 +158,17 @@ class StreamOutput:
     """Represents a stream output."""
 
     def __init__(
-        self, hass: HomeAssistant, idle_timer: IdleTimer, deque_maxlen: int = None
+        self,
+        hass: HomeAssistant,
+        idle_timer: IdleTimer,
+        deque_maxlen: int | None = None,
     ) -> None:
         """Initialize a stream output."""
-        self._hass = hass
-        self._idle_timer = idle_timer
+        self._hass: HomeAssistant = hass
+        self._idle_timer: IdleTimer = idle_timer
```

Comment:
This will be automatically inferred, same for `int`.
```suggestion
        self._hass = hass
        self._idle_timer = idle_timer
```